omnibus/files/private-chef-cookbooks/private-chef/recipes/bootstrap.rb

```diff
@@ -1,70 +1,59 @@
 #
-# Author:: Adam Jacob (<adam@opscode.com>)
-# Copyright:: Copyright (c) 2011 Opscode, Inc.
+# Author:: Adam Jacob <adam@chef.io>
+# Copyright:: Copyright (c) 2011-2015 Chef Software, Inc.
 #
 # All Rights Reserved
 #
 
-require 'securerandom'
-
 # Enterprise Addon Install
```

Comment:
While we removed the meat of the comment, this one is still out of date. We do zero addon install here.